flight/Modules/TxPID/txpid.c

```diff
@@ -48,6 +48,10 @@
 #include "accessorydesired.h"
 #include "manualcontrolcommand.h"
 #include "stabilizationsettings.h"
+#ifdef UAVOBJ_INIT_vtolpathfollowersettings
```

Comment:
It's needed on the other lines to not try and do things with objects that don't exist on non-navigation targets.  It could strictly be removed up there on that first one... but then we wouldn't get a compile time error if someone does something with vtolpathfollowersettings outside a similar #ifdef.  (It'd show up at link time instead and be a lot more confusing).